### PR TITLE
Task/Implement notification layer

### PR DESCRIPTION
**Pull request by TonyH277**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -8,8 +8,9 @@ spring.jpa.hibernate.ddl-auto=validate
 spring.jpa.show-sql=true
 spring.jpa.open-in-view=false
 
-server.servlet.context-path=/api
+telegram.bot.username=${BOT_NAME}
+telegram.bot.token=${BOT_TOKEN}
+telegram.bot.group.id=${GROUP_ID}
```
Comment (MolchanovAlexander) [inline src/main/resources/application.properties:13]: Suggest you to add TELEGRAM three times before its variables names.

Comment (MolchanovAlexander) [inline src/main/resources/application.properties:13]: Suggest you to add TELEGRAM three times before its variables names.

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -8,8 +8,9 @@ spring.jpa.hibernate.ddl-auto=validate
...
 
 jwt.expiration=${EXPIRATION_TIME}
 jwt.secret=${SECRET_KEY}
```
Comment (MolchanovAlexander) [inline src/main/resources/application.properties:16]: It is preferred to call them clearly.
jwt.expiration=${JWT_EXPIRATION}
jwt.secret=${JWT_SECRET}

**Review by MolchanovAlexander — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+# Final stage
+FROM openjdk:21-slim
+WORKDIR application
+COPY --from=builder application/dependencies/ ./
+COPY --from=builder application/spring-boot-loader/ ./
+COPY --from=builder application/snapshot-dependencies/ ./
+COPY --from=builder application/application/ ./
+ENTRYPOINT ["java", "org.springframework.boot.loader.launch.JarLauncher"]
+EXPOSE 8080
```
Comment (Nikname2303) [inline Dockerfile:16]: don't have last line, if you have it on your local version, just make indentation

**Review by Nikname2303 — CHANGES_REQUESTED** [review]
LGTM, just one minor comment

**Review by Nikname2303 — APPROVED** [review]

**Pull request merged**